From 33d3af1d354f2426eb666b1b0b5da76e065c2cc2 Mon Sep 17 00:00:00 2001
From: Devajith Valaparambil Sreeramaswamy
 <devajith.valaparambil.sreeramaswamy@cern.ch>
Date: Thu, 31 Oct 2024 09:32:20 +0100
Subject: [PATCH 37/51] Revert "Accessor to "is file opened"; this is crucial
 info for us."

This reverts commit 2558cf364e3c77d6373b88796b131e72aec266d8.
---
 interpreter/llvm-project/clang/include/clang/Basic/FileEntry.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/interpreter/llvm-project/clang/include/clang/Basic/FileEntry.h b/interpreter/llvm-project/clang/include/clang/Basic/FileEntry.h
index a9a05256dd..35efa14795 100644
--- a/interpreter/llvm-project/clang/include/clang/Basic/FileEntry.h
+++ b/interpreter/llvm-project/clang/include/clang/Basic/FileEntry.h
@@ -332,7 +332,6 @@ public:
   StringRef getName() const { return LastRef->getName(); }
 
   StringRef tryGetRealPathName() const { return RealPathName; }
-  bool isOpen() const { return (bool)File; }
   off_t getSize() const { return Size; }
   unsigned getUID() const { return UID; }
   const llvm::sys::fs::UniqueID &getUniqueID() const { return UniqueID; }
-- 
2.43.5

